lib/src/native/realm_core.dart

```diff
@@ -1338,6 +1338,12 @@ void _intoRealmValue(Object? value, Pointer<realm_value_t> realm_value, Allocato
         }
         realm_value.ref.type = realm_value_type.RLM_TYPE_UUID;
         break;
+      case DateTime:
+        final microseconds = (value as DateTime).toUtc().microsecondsSinceEpoch;
+        realm_value.ref.values.timestamp.seconds = microseconds ~/ 1000000;
+        realm_value.ref.values.timestamp.nanoseconds = (microseconds % 1000000) * 1000;
+        realm_value.ref.type = realm_value_type.RLM_TYPE_TIMESTAMP;
+        break;
```

Comment:
So no timezone :-/ .. Was about to suggest using https://pub.dev/packages/timezone, but since core doesn't support there is little point.

Reply:
Yeah, that's something we've never supported and it comes up occasionally as an annoyance.